agent/config.py

```
@@ -1,13 +1,17 @@
 import os
 import logging
 import secrets
+from pathlib import Path
 from dataclasses import dataclass, field
 from typing import List, Optional
```

Ruff (0.15.13)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)